source/mongorestore/mongorestore-behavior-access-usage.txt

```diff
@@ -0,0 +1,178 @@
+.. _mongorestore-behavior-access-usage:
+
+============================================
+mongorestore Behavior, Access, and Usage
```

Comment:
@nvillahermosa-mdb on the other pages we've split, this page was just <tool-name> Behavior, even though Access & Usage are in the page hierarchy.  I'd vote for keeping it to Behavior for simplicity's sake if you're good with that.